staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -314,8 +314,8 @@ func DefaultKubernetesUserAgent() string {
 // if called from a process not running in a kubernetes environment.
 func InClusterConfig() (*Config, error) {
 	const (
-		tokenFile  = "/var/run/secrets/kubernetes.io/serviceaccount/token"
-		rootCAFile = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
these correspond to the path used to mount the secret in admission:
https://github.com/kubernetes/kubernetes/blob/cde4c9ebe1a05738481781209ccfe14f99f6e7e3/plugin/pkg/admission/serviceaccount/admission.go#L51-L53

what are you expecting to mount to `C:/secrets/kubernetes.io/serviceaccount`?

Reply:
> Here we just copy the token and ca.crt from unix system to windows system. It is not produced from the windows system.

I'm not sure I understand this. If client-go is going to specify a fixed path it will try to load the token from, that needs to correspond with a path that will actually be populated by default.

Where does the automounted token currently appear within a windows filesystem?

Reply:
@liggitt Yes, your understanding is correct. Now, the token can not be automounted within a windows filesystem.

Reply:
No. We add the windows filesystem path, if I want to compile and project in windows system, then I can copy the related token to the specified path. Thanks for your respond.

Reply:
If you want to compile and run a client yourself, you can set up a rest.Config object pointing at any token file location you want, you don't need to modify the inclusterconfig path used by all windows clients. Modifying that path without also ensuring that is actually the location where tokens are mounted to windows containers isn't something we should do.